website/content/installation/_index.md

```diff
@@ -91,7 +101,7 @@ at the remote kustomization file :
 namespace: metallb-system
 
 resources:
-  - github.com/metallb/metallb//manifests?ref=v0.9.3
+  - github.com/metallb/metallb/manifests?ref=v0.9.3
```

Comment:
Uff, v0.9.3 is 2 years old now. Should this be updated to just point to `main` and so also be consistent with the change above (FRR manifests)?

```suggestion
  - github.com/metallb/metallb/manifests?ref=v0.9.3
```

Reply:
I'd say it's better to use the tag as an example.